src/reference/asciidoc/logging.adoc

```diff
@@ -181,3 +181,74 @@ public class MyEnhancedAppender extends AmqpAppender {
 
 }
 ----
+
+==== Customizing the Client Properties
+
+===== Simple String Properties
+
+Each appender supports adding client properties to the RabbitMQ connection.
+
+.log4j
+[source, text]
+----
+log4j.appender.amqp.clientConnectionProperties=foo:bar,baz:qux
```

Comment:
How about to add a raw for this new property in the `Common Appender Properties` table above? 
(I can do that on merge. Just let me know!)